CHANGELOG.rst

```diff
@@ -15,6 +15,9 @@ Next Release (TBD)
   (`#351 <https://github.com/awslabs/chalice/pull/351>`__)
 * Fix vendor directory contents not being importable locally
   (`#350 <https://github.com/awslabs/chalice/pull/350>`__)
+* Add support for binary payloads
+  (`#352 <https://github.com/awslabs/chalice/pull/352>`__)
```

Comment:
Crap, I should have caught this earlier, but you can all link to the proposals instead of the pull requests.  I think that's more useful for people browsing the changelog.